setup-local/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "setup-local",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

```suggestion
  "version": "1.0.1",
```
since we are just adding a patch.. and no new features/apis etc.
Similarly for setup-env...

---

Done